docs/README.md

```diff
@@ -2544,6 +2615,32 @@ source <path-to-your-bash-file> && boost deploy -e production
 
 Now just let the magic happen, Booster will create everything for you and give you back your app ready to use URL. 🚀
 
+##### Kubernetes provider
+
+To configure Kubernetes as a provider you need to meet certain prerequisites:
+
+- Config the Kubernetes cluster beforehand in a cloud provider or on-premises.
+- Install **kubectl**.
```

Comment:
We may rephrase this with something like: `install Kubectl and connect it to your Kubernetes cluster`

Reply:
ok